src/servers/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ endif() # TRTIS_ENABLE_GRPC
 set(HTTP_ENDPOINT_OBJECTS "")
 set(HTTP_ENDPOINT_LIBRARIES "")
 
-if(${TRTIS_ENABLE_HTTP} OR ${TRTIS_ENABLE_METRICS})
+if((${TRTIS_ENABLE_HTTP} OR ${TRTIS_ENABLE_METRICS}) AND (NOT ${TRTIS_ENABLE_HTTP_V2}))
   find_package(libevhtp CONFIG REQUIRED)
   message(STATUS "Using libevhtp ${libevhtp_VERSION}")
 
```

Comment:
Within here can just conditionalize HTTP_ENDPOINT_SRCS  on TRTIS_ENABLE_HTTP_V2. Don't need to duplicate everything.

Reply:
Done